contract_compassion/tests/test_contract_compassion.py

```diff
@@ -84,6 +85,19 @@ def test_contract_compassion_first_scenario(self):
             In this test we are testing states changement of a contract and if
             the old invoice are well cancelled when we pay one invoice.
         """
+        current_year = str(int(datetime.today().strftime(DF)[:4]) + 1)        
+        fiscal_year_id = self.registry('account.fiscalyear').create(
+            self.cr, self.uid, {
+                'name': current_year,
+                'code': current_year,
+                'date_start': str(datetime(

```

Comment:
It is not good to convert a date using str() method. You should use .strptime() method on date objects. Look for python documentation if needed.